experimental/backwards-compatibility/node16/package.json

```diff
@@ -19,6 +19,6 @@
   "author": "OpenTelemetry Authors",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=14"
+    "node": ">=18"
```

Comment:
we may want to remove that whole package (testing backwards compatibility for node 16 types)

Reply:
I kept them for now because nothing was broken and they're a template for testing `18` when we bump `@types/node` to `20`

Reply:
alright sounds good, let's keep it then :slightly_smiling_face: